### PR TITLE
OldAccount requirement

### LINKED ISSUE #332: Restrictions to s!rep command
There should be a system in place to prevent new discord users to give out rep. People with alt accounts can give rep to themselves and abuse the system. The same goes for the s!pay command. You basically get 2x the s!daily reward and transfer it to your other account with s!pay

### LINKED ISSUE #393: Add onlyOldsAccounts as command requeriment
rep and daily

### PR DESCRIPTION
**Pull request by Doges**
Closes #332 
Closes #393

### CODE REVIEW

**Comment by Doges** [comment]
missed rep, 1 sec

File: src/locales/en-US/errors.json [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
+  "invalidUser": "You need to give me a valid user.",
+  "invalidUserBot": "You need to give me a user that isn't a bot.",
+  "invalidUserNotBot": "You need to give me a user that is a bot.",
+  "databaseOnly": "Sorry, I don't have a database connection right now!",
+  "playerManagerOnly": "Sorry, I don't have a player connection right now!",
+  "invalidColor": "You need to give me a valid color.",
+  "invalidGuild": "You need to give me a valid guild.",
+  "invalidEmoji": "You need to give me a valid emoji.",
+  "userCantBeDeveloper": "User can't be a developer",
+  "onlyOldAccounts": "Sorry, you need to have at least 1 month old account to do that."
```
Comment (pedrofracassi) [inline src/locales/en-US/errors.json]:
```suggestion
  "onlyOldAccounts": "Your account must be at least one month old in order to do that"
```

File: src/commands/social/rep.js [hunk 2]
```diff
@@ -10,6 +10,8 @@ module.exports = class Rep extends Command {
     this.name = 'rep'
     this.category = 'social'
 
+    this.requirements = new CommandRequirements(this, { onlyOldAccounts: true, databaseOnly: true })
```
Comment (pedrofracassi) [inline src/commands/social/rep.js]: I love how this didn't have databaseOnly already

Comment (Doges) [reply src/commands/social/rep.js]: HEUHEUEHUEHEUHEU

**Pull request merged**